doc/examples/jupyter_notebook/readme.md

```diff
@@ -0,0 +1,49 @@
+# Jupyter Notebook using versioned Pachyderm Data
+
+In the following example, we will explore attaching a Jupyter server to Pachyderm and manipulating data in Pachyderm's data versioning.  Specifically, we will attach our Jupyter server to three different repositories used in an example Pachyderm pipeline:
```

Comment:
Instead of 'in Pachyderm's data versioning' can we say something like 'and manipulating versioned data sets using Pachyderm'